samples/link-unfurling-stt/nodejs/bots/teamsBot.js

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+const { TeamsActivityHandler, CardFactory } = require("botbuilder");
+
+class TeamsBot extends TeamsActivityHandler {
+    constructor() {
+        super();
+        this.onMembersAdded(async (context, next) => {
+            const membersAdded = context.activity.membersAdded;
+            for (let member = 0; member < membersAdded.length; member++) {
+                if (membersAdded[member].id !== context.activity.recipient.id) {
+                    await context.sendActivity("Hello and welcome! This sample demonstrates the use of link unfurling in share to teams scope.");
+                }
+            }
+
+            await next();
+        });
```

Comment:
We don't have Bot mentioned in manifest, could you please remove code which is not being used?